pkg/channel/distributed/common/kafka/sarama/sarama_test.go

```diff
@@ -164,10 +165,12 @@ func TestEnableSaramaLogging(t *testing.T) {
 	}()
 
 	// Perform The Test
-	EnableSaramaLogging()
+	EnableSaramaLogging(true)
 
 	// Verify Results (Not Much Is Possible)
 	sarama.Logger.Print("TestMessage")
+
+	EnableSaramaLogging(false)
 }
 
```

Comment:
print again after disabling? (i know it's just visual verification but better than nothing ; )

Reply:
done